Bottom: 0352c3b90a2631e5e402a614f4d910f305c6c3d3
Top:    01a94df52b4ff33876c1d7340d1328b024b4f3eb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 01:52:48 +0000

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index e206cea..b0a5049 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -38,7 +38,6 @@
 #include "hidgl.h"
 
 
-
 #ifdef HAVE_LIBDMALLOC
 #include <dmalloc.h>
 #endif
